androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivityMetadata.java

```diff
@@ -126,12 +140,15 @@
             "android.support.customtabs.trusted.SCREEN_ORIENTATION";
 
     private final static int DEFAULT_COLOR_ID = android.R.color.white;
+    private final static int DEFAULT_DIVIDER_COLOR_ID = 0x00000000;
```

Comment:
Isn't `0x00000000` a colour, not a colour id? If you try to load from id 0, won't we get a crash?

There apparently is `android.R.color.transparent`.

Reply:
As an additional note, `0` is not a valid resource Id. See https://developer.android.com/reference/android/content/res/Resources.html#getIdentifier(java.lang.String,%20java.lang.String,%20java.lang.String)